docs/orm-integration-to-generic.md

```diff
@@ -0,0 +1,310 @@
+The AEA generic seller with ORM integration demonstrate how to interact with a database using python-sql objects.
```

Comment:
Why does this need to be a separate guide. Imo it should be an extra in the generic guide.

Reply:
Otherwise A LOT of repeat explanations

Reply:
Well the idea is that we have a use case that shows how to get and sell data from the `yaml` file and then we step on this guide and we increase the complexity by modifying the previous agent. And I would like to keep the "simple" guide as clean as possible.

Reply:
Yes, but surely the simple guide could just have a section at the end with the additional work needed. Or you reference the simple guide here. Duplication is a bit suboptimal